macros/g4simulations/G4Setup_fsPHENIX.C

```diff
@@ -0,0 +1,286 @@
+int Min_si_layer = 1;
+int Max_si_layer = 6;
+int Min_cemc_layer = 1;
+int Max_cemc_layer = 30;
+int Min_hcal_in_layer = 1;
+int Max_hcal_in_layer = 1;
+int Min_hcal_out_layer = 1;
+int Max_hcal_out_layer = 1;
+
+double no_overlapp = 0.0001; // added to radii to avoid overlapping volumes
+bool overlapcheck = false; // set to true if you want to check for overlaps
+
+void G4Init(bool do_svtx = true,
+	    bool do_preshower = false,
+	    bool do_cemc = true,
+	    bool do_hcalin = true,
+	    bool do_magnet = true,
+	    bool do_hcalout = true,
+	    bool do_pipe = true,
+	    bool do_bbc = true,
+	    bool do_global = true,
+	    bool do_EEMC = true,
+	    bool do_FEMC = true,
+	    bool do_FHCAL = true) {
+
+  gROOT->LoadMacro("G4_Bbc.C");
+  if (do_bbc) BbcInit();
+  
+  gROOT->LoadMacro("G4_Pipe.C");
+  if (do_pipe) PipeInit();
+  
+  // load detector macros and execute Init() function
+  gROOT->LoadMacro("G4_Svtx.C");           // default
+  //gROOT->LoadMacro("G4_Svtx_ladders.C"); // testing
+  //gROOT->LoadMacro("G4_Svtx_ITS.C");     // testing
+  if (do_svtx) SvtxInit();
+
+  if (do_preshower) {
+    gROOT->LoadMacro("G4_PreShower.C");   // testing
+    PreShowerInit();
+  }
+
+  gROOT->LoadMacro("G4_CEmc_Spacal.C");    //
+  //gROOT->LoadMacro("G4_CEmc_cross.C");   // obselete
+  //gROOT->LoadMacro("G4_CEmc_Alice.C");   // obselete
+  //gROOT->LoadMacro("G4_CEmc_Alice_W.C"); // obselete
+  if (do_cemc) CEmcInit(Cemc_slats_per_cell);
+
+  //gROOT->LoadMacro("G4_Hcal_ref.C");     // deprecated
+  //gROOT->LoadMacro("G4_Hcal_cross.C");   // deprecated
+
+  gROOT->LoadMacro("G4_HcalIn_ref.C");       // default 
+  if (do_hcalin) HCalInnerInit();
+
+  gROOT->LoadMacro("G4_Magnet.C");
+  if (do_magnet) MagnetInit();
+
+  gROOT->LoadMacro("G4_HcalOut_ref.C");       // default 
+  if (do_hcalout) HCalOuterInit();
+
+  gROOT->LoadMacro("G4_Global.C");
+  
+  gROOT->LoadMacro("G4_Jets.C");
+
+  gROOT->LoadMacro("G4_FwdJets.C");
```

Comment:
Good point. I was using a modified version of G4_Jets.C. I think having a separate G4_FwdJets.C is a good idea.  I can certainly add one.